charts/applicationset-progressive-sync/crds/argoproj.skyscanner.net_progressivesyncs.yaml

```diff
@@ -0,0 +1,237 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.4.1
+  creationTimestamp: null
+  name: progressivesyncs.argoproj.skyscanner.net
+spec:
+  group: argoproj.skyscanner.net
+  names:
+    kind: ProgressiveSync
+    listKind: ProgressiveSyncList
+    plural: progressivesyncs
+    singular: progressivesync
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: ProgressiveSync is the Schema for the progressivesyncs API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: ProgressiveSyncSpec defines the desired state of ProgressiveSync
+            properties:
+              sourceRef:
```

Comment:
Yes, I'm going to raise a separate PR where I update the chart because otherwise the chart test breaks. 
This is because `ct` tries to pull an actual image.